src/FSLibrary/StellarCoreHTTP.fs

```diff
@@ -179,14 +171,7 @@ type LoadGen =
           skiplowfeetxs = false
           wasms = None
           instances = None
-          dataEntriesLow = None
```

Comment:
These configs are part of loadgen configs, why do we remove them here?

Reply:
These configs aren't HTTP options anymore. All of these became distributions rather than low/high values, and in doing so I moved them to the config file instead of using HTTP options. They're now declared as config file options in `Program.fs`

Reply:
Ah yes, makes sense. Sorry, I think I reviewed this change before ☕  :)